ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -138,6 +138,24 @@ tests:
       EC2_AMI: ami-0c231be9fb222349e
       EC2_INSTANCE_TYPE: a1.2xlarge
     workflow: openshift-microshift-e2e-openshift-conformance-reduced-aws
+- as: e2e-reboot
+  cluster: build02
+  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
+  steps:
+    cluster_profile: aws
+    env:
+      EC2_AMI: ami-07f5ac8ddbcfbb96e
+    workflow: openshift-microshift-e2e-reboot-tests-aws
+- as: e2e-reboot-arm
+  cluster: build02
+  optional: true
+  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
+  steps:
+    cluster_profile: aws
+    env:
+      EC2_AMI: ami-0c231be9fb222349e
+      EC2_INSTANCE_TYPE: a1.2xlarge
+    workflow: openshift-microshift-e2e-reboot-tests-aws
```

Comment:
This is already part of the e2e test defined [here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/microshift/openshift-microshift-main.yaml#L141-L149).